.github/CODEOWNERS

```diff
@@ -0,0 +1,32 @@
+# See https://github.com/scottrigby/prometheus-helm-charts/issues/12
+# https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+# https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
+
+# The repo admins team will be the default owners for everything in the repo.
+# Unless a later match takes precedence, they will be requested for review when someone opens a pull request.
+* @NethermindEth/Angkor/angkor-validator-operators
+
+/.github/workflows/ @NethermindEth/Angkor/angkor-validator-operators @AntiD2ta @aivarasko @matilote
+
+/charts/cert-manager/ @Anish @Manjeet
+/charts/common/ @aivarasko @matilote
+/charts/dirk/ @aivarasko @matilote
+/charts/ethereum-node/ @Anish @Manjeet
+/charts/execution-beacon/ @aivarasko @AntiD2ta @matilote
+/charts/external-dns/ @Anish @Manjeet
+/charts/external-secrets/ @Anish @Manjeet
+/charts/juno-node/ @Anish @Manjeet
+/charts/kong/ @Anish @Manjeet
+/charts/kube-prometheus-stack/ @Anish @Manjeet
+/charts/lodestar/ @aivarasko @matilote
```

Comment:
put Denis and Marcos there

Reply:
Done [d09b28b](https://github.com/NethermindEth/charts/pull/91/commits/d09b28bf604be0344158a63254d61c377ee6b433)